src/OrbitGl/ManualInstrumentationManager.cpp

```diff
@@ -63,7 +63,14 @@ void ManualInstrumentationManager::ProcessStringEventLegacy(const orbit_api::Eve
 
 void ManualInstrumentationManager::ProcessStringEvent(
     const orbit_client_protos::ApiStringEvent& string_event) {
-  string_manager_.AddOrReplace(string_event.async_scope_id(), string_event.name());
+  // A string can be sent in chunks so we append the current value to any existing one.
```

Comment:
I guess that is not true anymore with the encoded_string change no?

Reply:
I decided to keep that, as the old strings are much easier to handle that way. So we can simply turn the old chunks into new chunks.

This will also allow us to write the converter for the grpc protos more easily, as it does not need to first collect all chunks.

And, in theory users could already rely on the append behaviour, when putting two string tracking macros next to each other with same id.